src/main/java/uk/gov/hmcts/reform/professionalapi/controller/external/ProfessionalExternalUserController.java

```diff
@@ -78,14 +82,71 @@
     @Secured({"pui-finance-manager", "pui-user-manager", "pui-organisation-manager", "pui-case-manager", "caseworker-divorce-financialremedy", "caseworker-divorce-financialremedy-solicitor", "caseworker-divorce-solicitor", "caseworker-divorce", "caseworker"})
     public ResponseEntity<?> findUsersByOrganisation(@ApiParam(hidden = true) @OrgId String organisationIdentifier,
                                                      @ApiParam(name = "showDeleted", required = false) @RequestParam(value = "showDeleted", required = false) String showDeleted,
-                                                     @ApiParam(name = "email", required = false) @RequestParam(value = "email", required = false) String email,
```

Comment:
Why are we removing email address from this REST endpoint? Was this something that was agreed by Sourav/The business? Also your Jira-531 is for addressing QA concerns which does NOT seem to be the content of this Jira?
Please explain.

Reply:
This was a "Critical" Sonar Issue that was raised, I am unsure if it was a business agreement but it was discussed with Shreedhar